### PR TITLE
feat(template): Adding McCabe to the template.

### PR DESCRIPTION
**Pull request by Anavelyz**
## Pull Request description
Adding McCabe to the template.

Solving issue #35 

## Pull Request checklists

This PR is a:
- [ ] bug-fix
- [x] new feature
- [ ] maintenance

About this PR:
- [x] it includes tests.
- [ ] the tests are executed on CI.
- [ ] the tests generate log file(s) (path).
- [x] pre-commit hooks were executed locally.
- [ ] this PR requires a project documentation update.

Author's checklist:
- [x] I have reviewed the changes and it contains no misspelling.
- [ ] The code is well commented, especially in the parts that contain more complexity.
- [x] New and old tests passed locally.

## Reviewer's Checklist

- [ ] I managed to reproduce the problem locally from the `main` branch
- [ ] I managed to test the new changes locally
- [ ] I confirm that the issues mentioned were fixed/resolved

### CODE REVIEW

File: {{cookiecutter.project_slug}}/.pre-commit-config.yaml [hunk 1]
```diff
@@ -111,7 +111,7 @@ repos:
     {% if cookiecutter.use_mccabe in ['yes', 'y'] -%}
     -   id: mccabe
         name: mccabe
-        entry: mccabe --max-complexity=5
+        entry: python -m mccabe --min 5 *.py
```
Comment (Anavelyz) [inline {{cookiecutter.project_slug}}/.pre-commit-config.yaml]: @xmnlab Do you have any idea why this is not working?

Comment (xmnlab) [reply {{cookiecutter.project_slug}}/.pre-commit-config.yaml]: this is what I found in the documentation `python -m mccabe --min 5 mccabe.py` ... so it should work .. what is the error message you have?

Comment (Anavelyz) [reply {{cookiecutter.project_slug}}/.pre-commit-config.yaml]: Executable `mccabe` not found

https://github.com/osl-incubator/cookiecutter-python/actions/runs/5075606779/jobs/9116994061#step:5:369

Comment (xmnlab) [reply {{cookiecutter.project_slug}}/.pre-commit-config.yaml]: not sure tbh .. I can give you a hand on that tomorrow

Comment (Anavelyz) [reply {{cookiecutter.project_slug}}/.pre-commit-config.yaml]: A thousand thanks!

For now, I implemented it with the flake8 plugin and it worked.

**Review by xmnlab — APPROVED** [review]
LGTM! Nice docs!

**Pull request merged**